src/Mirage-Carousel/MICarouselSettings.class.st

```diff
@@ -7,12 +7,13 @@ Class {
 	#classInstVars : [
 		'thumbnailUpdateTime'
 	],
-	#category : 'Mirage-Carousel-Settings'
+	#category : #'Mirage-Carousel-Settings'
 }
 
 { #category : #accessing }
 MICarouselSettings class >> defaultThumbnailUpdateTime [
-	^ 5 seconds
```

Comment:
Why did you removed the instantiation of a duration from here?

Reply:
I stumbled with this bug opening the settings after loading the  `dev` branch on Pharo 7.

A debugger opens while trying to scale the default value (5 seconds duration) using an integer range from 1 to 30 configured on `MICarouselSettings class>>#wpThumbnailUpdateTimeSettingsOn:`. 

The `Duration>#roundTo:` expects another duration as the collaborator and I spent a while trying to fix it keeping the duration and, after failing miserably, I finally changed it for an integer.

Reply:
Ok, seems legit.